kratos/tests/processes/test_replace_elements_and_condition_process.cpp

```diff
@@ -0,0 +1,257 @@
+//    |  /           |
```

Comment:
this file should not be modified (or added) in this PR, since the replace function should also not be changed

Reply:
There is a PR specific for this for the master

#2221

Reply:
It is exactly the same, but with a test added

Reply:
I am accepting this since we are late for the release. next time i will not allow backporting a feature that is not needed.